website/app/controllers/show.js

```diff
@@ -164,8 +164,6 @@ export default class ShowController extends Controller {
     if (this.tabs.length > 1 && this.currentActiveTabIndex != 0) {
       // for consistency we always set the query param tab to a lowercase version of the tab label
       set(this, 'selectedTab', this.tabs[current].label.toLowerCase());
-    } else {
-      set(this, 'selectedTab', null);
```

Comment:
this _was_ helpful to cleanup the URL is someone were to link to a query param that didn't exist, but in practice it does still work the same 🤔

Reply:
yep, it still defaults to first tab, but doesn't clear the query param